_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,237 @@
+---
+### orinally sourced from https://www.reddit.com/r/Bitcoin/comments/3urm8o/optin_rbf_is_misunderstood_ask_questions_about_it/
+title: Opt-in RBF FAQ
+name: optin-rbf-faq
+type: page
+permalink: /en/faq/optin_rbf
+share: true
+---
+{% include _toc.html %}
+
+##What is RBF?
+
+RPF or replace by fee is a way of enabling pending transactions to be modified -- or double spent.
+
+Opt-in RBF means transactions can be marked double spendable or not.
```

Comment:
One thing I've noticed was that users don't really seem to fully understand the "opt-in" part of Opt-in RBF. The section might be split into "What is RFB" and "What is Opt-In RBF" to highlight the difference, and that Opt-in RBF isn't a global thing, but rather something based on a transaction-per-transaction basis.